recipes/scikit-geometry/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "scikit-geometry" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/scikit-geometry/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 89f7d3a6795d657cdf75b0c3d20d8929765e4ea8eae6da6a61edb538c9664603
+  patches:
+    - osx_cpp14.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python 3.*
```

Comment:
It might be that this causes errors in the feedstock, then you also might want to add `skip: true  # [py2k]` later.